src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -168,31 +168,33 @@
 
 (defn- skip-load?
   [name macros?]
-  ((if macros?
-     '#{cljs.core
+  (if macros?
+    ('#{cljs.core
```

Comment:
A `contains?` would probably make this more readable

Reply:
Pr welcome